charts/jaeger/README.md

```diff
@@ -220,182 +236,3 @@ helm install jaeger jaegertracing/jaeger \
   --set storage.kafka.brokers={<BROKER1:PORT>,<BROKER2:PORT>} \
   --set storage.kafka.topic=<TOPIC>
 ```
-
-## Configuration
```

Comment:
Was the entire configuration block meant to be deleted? There are a lot more configuration than the Cassandra you removed.

Reply:
Yes, https://helm.sh/docs/helm/helm_show_values/ will remove the necessity to duplicate these in the README, point taken about sub chart dependencies I will add a note about how to retrieve their values.

Reply:
Is this a standard practise now to remove the configuration from documentation though? Still seems very handy to be able to see what options can be set. Considering the organisation I work for is still on Helm 2, hard to know what the current best practises are so hopefully others can assist here.

Reply:
Yes that seems to be the direction charts are moving towards. I take it your org will be moving to helm3 in the near future considering the deprecation timeline?

@scottrigby Could you please chime in?

Reply:
Also https://github.com/jaegertracing/helm-charts/pull/166/commits/8cc7aa84ad48f7d32e419b7e8fc7bc7b2bc74355 pushed to address dependency chart values

Reply:
Great, thanks @scottrigby. As such I will include the helm 2 equivalent commands. That work for you @mikelorant ?

Reply:
Works for me.

Reply:
Ok great, I've updated the README accordingly.